test/unit/insights.test.js

```diff
@@ -27,14 +27,19 @@ describe('Application Insights', () => {
 
   test('does not setup application insights if no instrumentation key', () => {
     const appInsights = require('../../app/insights')
+    delete process.env.APPINSIGHTS_INSTRUMENTATIONKEY
```

Comment:
If you don't ensure the env var is empty, then the test behaviour will depend on on whether the user has that env var set locally or not